embc-app/ClientApp/src/app/volunteer/pages/referral-maker/referral-maker.component.html

```diff
@@ -42,19 +42,15 @@ <h1>Identify Supports</h1>
             <div class="col">
               <strong>Valid From</strong>
             </div>
-            <div class="col">
-              <strong>Valid To</strong>
-            </div>
-            <div class="col-2">
-
-            </div>
           </div>
           <div class="row">
             <div class="col">
               <app-date-time-picker [(value)]="workingDefaultDate"></app-date-time-picker>
             </div>
             <div class="col-2">
-              <button type="button" class="btn btn-secondary float-right" (click)="defaultDate=workingDefaultDate">UPDATE</button>
+              <button type="button" class="btn btn-primary float-right text-uppercase" (click)="defaultDate=workingDefaultDate; !defaultDate? showDefaultDatePicker=true:showDefaultDatePicker=false;">Update</button>
```

Comment:
I recommend this be changed to a function in the TS file.
IMHO, it's OK to put some minimal functionality (code) in the presentation file (HTML), but in this case it's more than a trivial amount of code.